inference-engine/samples/common/utils/src/args_helper.cpp

```diff
@@ -326,3 +356,94 @@ void printInputAndOutputsInfo(const InferenceEngine::CNNNetwork& network) {
                   << layer.second->getLayout() << std::endl;
     }
 }
+
+void configurePrePostProcessing(std::shared_ptr<ov::Function>& function,
+                                const std::string& ip,
+                                const std::string& op,
+                                const std::string& iop,
+                                const std::string& il,
+                                const std::string& ol,
+                                const std::string& iol) {
+    auto preprocessor = ov::preprocess::PrePostProcessor();
```

Comment:
Will this preprocessing be applied in the `PrePostProcessor` destructor? Or we are missing some `apply` call?

Reply:
we need `apply` call

Reply:
Yes I forgot to call build 😄